Lesson7_OOPS.cpp

```diff
@@ -0,0 +1,106 @@
+/* DESIGN A VENDING MACHINE */
+#include<iostream>
+using namespace std;
+class Inventory
+{
+	int coke,sprite,chips;
+	static int coke_count,sprite_count,chips_count;
```

Comment:
do all inventories share coke_count, sprite_count, and chips_count?  Static makes this value shared between different instances of inventory, which doesn't seem right